R/transformCounts.R

```diff
@@ -56,6 +56,36 @@
 #' 'relabundance', 'rrank', 'standardize', 'total': please refer to 
 #' \code{\link[vegan:decostand]{decostand}} for details.
 #' 
+#' \item 'css': Cumulative Sum Scaling (CSS) can be used to normalize count data 
+#' by accounting for differences in library sizes. [Paulson et al. (2013) Differential abundance analysis for microbial marker-gene surveys]
+#' (https://doi.org/10.1038/nmeth.2658) outlines this in more detail. 
```

Comment:
Could we just cite here and put the more exact literature reference in the @reference section?